src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -44,23 +49,27 @@ object CakeBuildPlugin extends AutoPlugin {
     // WORKAROUND DockerPlugin doesn't like '+'
     version := version.value.replace('+', '-'),
     concurrentRestrictions := {
-      val limited = Properties.envOrElse("SBT_TASK_LIMIT", "4").toInt
+      val limited = sys.env.get("SBT_TASK_LIMIT").flatMap(n => Try(n.toInt).toOption).getOrElse(4)
       Seq(Tags.limitAll(limited))
     }
   )
 
-  override val projectSettings = Seq(
+  /** @inheritdoc */
+  override val projectSettings: Seq[Setting[_]] = Seq(
     resolvers += Resolver.bintrayRepo("cakesolutions", "maven"),
     ivyLoggingLevel := UpdateLogging.Quiet,
     conflictManager := ConflictManager.strict,
-    // makes it really easy to use a RAM disk
+    // makes it really easy to use a RAM disk - when it exists, the
+    // SBT_VOLATILE_TARGET directory is created as a side effect when setting
```

Comment:
this isn't just created by javaOptions it's also created by sbt... you don't have to create `target` directories, do you?